src/components/blog/SinglePostPage.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./singlePostPage.scss";
+
+export default function SinglePostPage(props) {
+  const postProps = props.location.state;
+  const title = postProps.title.rendered;
+  const date = postProps.date.split("T")[0];
```

Comment:
needs a comment here for the random T so reader understands. also i would consider calling this dateString since it's a string, usually date suggests its a JS Date type